examples/with-vite-react/README.md

```diff
@@ -1,6 +1,6 @@
 # `Turborepo` Vite starter
 
-This is an official starter Turborepo.
+This is a community-maintained example. If you experience a problem, please submit a pull request with a fix. GitHub Issues will be closed.
```

Comment:
Unrelated. I just happened to notice it and fixed. This example is not core-maintained.